modules/common/src/main/resources/reference.conf

```diff
@@ -280,3 +280,17 @@ surge {
   dr-standby-enabled = false
   dr-standby-enabled = ${?SURGE_ENABLE_DR_STANDBY}
 }
+
+prober {
```

Comment:
Maybe `execution-context-prober` or something on the naming here?

Reply:
Yes, you are right execution-context-prober sounds better ;-) I'm going to make the change.